ui/src/app/lib/atlasmap-data-mapper/components/document.definition.component.ts

```diff
@@ -103,9 +103,21 @@ export class DocumentDefinitionComponent {
     private scrollTop = 0;
     private searchResultsExist = false;
 
+    private sourcesTargetsLabel: String;
```

Comment:
`String` is not a valid TypeScript type, but a reference to JavaScript's underlying class type for... strings. That is why the compiler can honor this annotation although not correct. You should use `string` (all in lowercase) instead.

Reply:
Ugh good catch!! my hand is cursed by Java...